sample-build-playbook.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: "Run initial minimum requirement checks"
+  hosts: localhost
+  remote_user: root
+  tasks:
+    -
+      name: "Check Java is installed on path of local machine"
+      command: which java
+      connection: local
+
+- name: "Install Jenkins"
```

Comment:
We could group all these together and test on the buildfarm_platform variable to decide if android/ios are to be run.
e.g.
```
- name: "Deploy Digger"
  hosts: master
  remote_user: root
  roles:
    - {role: digger-installer/android-sdk, when: "'android' in {{ digger_platforms }}", tags: android_sdk}
    - {role: digger-installer/deploy-jenkins, tags: deploy-jenkins }
    - {role: digger-installer/configure-buildfarm, tags: configure-buildfarm}
  vars_files:
    - "vars/buildfarm.yml"
```